dev_tools/bash_scripts_test.py

```diff
@@ -208,7 +208,7 @@ def test_pytest_changed_files_branch_selection():
                  setup='mkdir alt\n'
                        'cd alt\n'
                        'git init --quiet\n'
-                       'git commit -m test --quiet --allow-empty\n'
+                       'git commit -m tes --quiet --allow-empty --no-gpg-sign\n'
```

Comment:
tes -> test

Reply:
Nope, that's on purpose to avoid wrapping the line. It's completely pointless text regardless.

Reply:
Sure.  I would have chosen 't' in this case.